src/PowerShell/Microsoft.WinGet.Client/README.md

```diff
@@ -0,0 +1,25 @@
+# Microsoft.WinGet.Client
+
+This is the source code for the PowerShell module for the Windows Package Manager Client.
+
+## Building
+### Pre-Requisite Steps
+
+A **PowerShell module** is merely a folder containing a **.psd1** file or a
+**.psm1** file. In this scenario, we will be building a folder with a **.psd1**
+file and all of the other necessary bits inside. But, before we do that, we need
+to make sure of the following steps.
+
+1. The project has been built for the **x64**, **x86**, **ARM64**, and **ARM**
+   platforms in **Release** mode.
+   - The build script also relies on the exact location and structure of the
+     actual build files. That is, the build script will look for a **bin**
+     directory in this folder and expects a directory structure like:
+     - `bin\x64\Release\net6.0-windows10.0.22000.0\`
+     - `bin\x86\Release\net461\`
+
+### Running the Script
+Once you've done all of the pre-requisite steps, you can build the PowerShell
+module folder by running the PowerShell build script. For example,
+
+- `.\BuildModule.ps1 -In bin -Out build\bin -Configuration Release`
```

Comment:
Can this be added to the build pipeline?

Reply:
Agreed, it would be great if that were done and the directory published as an artifact so that it was fairly easy to test from any given build.  I think that might require custom work though given that the current pipeline involves splitting the jobs by architecture (?).

Reply:
I've made a first pass at adding it to the build pipeline.